docs/DeveloperGuide.adoc

```diff
@@ -1224,3 +1258,193 @@ _{ more test cases ... }_
 .. _{explain how to simulate a missing/corrupted file and the expected behavior}_
 
 _{ more test cases ... }_
+
+//tag::favTest[]
```

Comment:
Seems like all test cases are covered with your manual testing test cases.

Reply:
Hmm, I think we can leave it up to the testers to vary their test inputs. Maybe it is best if we just provide suggestions to guide them on how to test.